index.js

```diff
@@ -139,21 +140,20 @@ function makePool (redisPool, database) {
                     no_ready_check: redisPool.options.noReadyCheck
                 });
 
-                client.on('error', function (err) {
-                    redisPool._log({ db: database, action: 'error', err: err.message });
```

Comment:
I we are not going to use it here, I would receive `redisPool.options` instead of `redisPool` in `makePool` function

Reply:
But I am not sure about removing it. I supposed a client can be adquired and after that, being errored for some reason. This event handler will catch it, end and flush connection. And we are going to lose that log.

Reply:
Yeah, you are right, we might want to keep logging here.

Reply:
Done